rllib/utils/test_utils.py

```diff
@@ -42,8 +45,8 @@ def framework_iterator(
     as the correct eager/non-eager contexts for tfe/tf.
 
     Args:
-        config: An optional config dict to alter in place depending on the
-            iteration.
+        config: An optional config dict or TrainerConfig object to alter in place
```

Comment:
I'm not sure I understand this sentence, sorry. What's altered in place and why?

Reply:
done